src/index.ts

```diff
@@ -2,7 +2,7 @@ import { DebugHandler, DebugConsume } from "./debugging";
 import * as en from "./locales/en";
 import { Chrono, Parser, Refiner } from "./chrono";
 import { ParsingResult } from "./results";
-import { Component, ParsedResult, ParsingOption, ParsingReference, Meridiem, Weekday } from "./types";
+import { Component, ParsedComponents, ParsedResult, ParsingOption, ParsingReference, Meridiem, Weekday } from "./types";
 
 export { en, Chrono, Parser, Refiner, ParsingResult };
```

Comment:
`ParsedComponents` needs to be exported here.

Reply:
Done! Sorry, that's what I get for not cloning and opening it in the IDE... 🤦🏻